lib/coingecko.ts

```diff
@@ -1,11 +1,12 @@
-import { onMainnet } from './chainEnv';
+import { SupportedNetwork } from './config';
 
 export async function getUnitPriceForCoin(
   tokenAddress: string,
   toCurrency: string,
+  network?: SupportedNetwork,
 ): Promise<number | undefined> {
-  if (!onMainnet) {
-    return 1.01;
+  if (network !== 'ethereum') {
+    return undefined;
```

Comment:
why undefined instead of 1.01?

Reply:
again prepping for optimism/polygon. can include a specific condition for rinkeby here